packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -927,6 +931,10 @@ class AccountInternal extends PureComponent<
   }
 
   getFilteredAmount = async () => {
+    if (!this.paged) {
+      return 0;
+    }
+
     const { data: amount } = await runQuery(
       this.paged?.query.calculate({ $sum: '$amount' }),
```

Comment:
We can probably remove the optional here
```suggestion
      this.paged.query.calculate({ $sum: '$amount' }),
```